pkg/csi/service/vanilla/controller_helper.go

```diff
@@ -60,26 +57,15 @@ func validateVanillaControllerUnpublishVolumeRequest(ctx context.Context,
 // validate ExpandVolumeRequest for Vanilla CSI driver.
 // Function returns error if validation fails otherwise returns nil.
 func validateVanillaControllerExpandVolumeRequest(ctx context.Context,
-	req *csi.ControllerExpandVolumeRequest, isOnlineExpansionEnabled, isOnlineExpansionSupported bool) error {
+	req *csi.ControllerExpandVolumeRequest, isOnlineExpansionSupported bool) error {
 	log := logger.GetLogger(ctx)
-	if err := common.ValidateControllerExpandVolumeRequest(ctx, req); err != nil {
-		return err
+	// Check online extend vCenter support.
+	if !isOnlineExpansionSupported {
```

Comment:
This error is getting double wrapped. Can you move code from 62:66 to ControllerExpandVolume function?